src/dft.cpp

```diff
@@ -985,6 +985,60 @@ complex<double> dft_chunk::process_dft_component(int rank, direction *ds, ivec m
   return integral;
 }
 
+// get variables that are needed by complex<double> fields::process_dft_component
+size_t *fields::get_dims(dft_chunk **chunklists, int num_chunklists, component c, ivec &min_corner, ivec &max_corner, size_t &array_size, size_t &bufsz, int &rank, direction *ds, int *array_rank, size_t *array_dims, direction *array_dirs){
+  /***************************************************************/
+  /* get statistics on the volume slice **************************/
+  /***************************************************************/
+  volume *where = &v; // use full volume of fields
+  bufsz = 0;
+  min_corner = gv.round_vec(where->get_max_corner()) + one_ivec(gv.dim);
+  max_corner = gv.round_vec(where->get_min_corner()) - one_ivec(gv.dim);
```

Comment:
You can just use `v.get_max_corner()` etcetera and eliminate `where` completely?